### ISSUE TITLE
fastfetch 1.12.1 reports incorrect Host on M2 Mac Studio with M2 Max CPU

### ISSUE THREAD

**Issue by lgw4** [fastfetch-cli/fastfetch#490]
# General description of bug:

* What happened: `fastfetch` reports `Host: Mac Studio (M2 Ultra, 2023, Two Thunderbolt 4 front ports)`
* What should happen: `fastfetch` should report `Host: Mac Studio (M2 Max, 2023, Two USB C front ports)`
* Did it work in an older version: No, only reported model identifier
* Where did you get the binary: macOS Homebrew
* Does this issue still occurs in [the latest dev build](https://github.com/fastfetch-cli/fastfetch/actions/)? Unknown. Dev builds are unsigned.

# Often helpful information:

Output of `fastfetch --version`:
```
fastfetch 1.12.1 (aarch64)
```

The content of the configuration file you use (if any)
```
no config file
```

Output of `env NO_CONFIG=1 fastfetch --load-config all --show-errors --stat --multithreading false --disable-linewrap false --hide-cursor false`:

```
                     c.'          xxxx@defiant                                                                                                            0ms
                 ,xNMM.           ------------                                                                                                            0ms
               .OMMMMo            OS: macOS Ventura 13.4.1 22F2083 arm64                                                                                  0ms
               lMM"               Host: Mac Studio (M2 Ultra, 2023, Two Thunderbolt 4 front ports)                                                        0ms
     .;loddo:.  .olloddol;.       Bios: iBoot-8422.121.3.1.1                                                                                              1ms
   cKMMMMMMMMMMNWMMMMMMMMMM0:     Board: Not supported on this platform                                                                                   0ms
 .KMMMMMMMMMMMMMMMMMMMMMMMWd.     Chassis: Not supported on this platform                                                                                 0ms
 XMMMMMMMMMMMMMMMMMMMMMMMX.       Kernel: 22.5.0                                                                                                          0ms
;MMMMMMMMMMMMMMMMMMMMMMMM:        Uptime: 6 days, 19 hours, 50 mins                                                                                       0ms
:MMMMMMMMMMMMMMMMMMMMMMMM:        Processes: 676                                                                                                          0ms
.MMMMMMMMMMMMMMMMMMMMMMMMX.       Packages: 225 (brew), 59 (brew-cask)                                                                                    0ms
 kMMMMMMMMMMMMMMMMMMMMMMMMWd.     Shell: fish 3.6.1                                                                                                       5ms
 'XMMMMMMMMMMMMMMMMMMMMMMMMMMk    Display (external): 5120x2880 @ 60Hz (as 2560x1440)
  'XMMMMMMMMMMMMMMMMMMMMMMMMK.    Display (external): 5120x2880 @ 60Hz (as 2560x1440)                                                                    46ms
    kMMMMMMMMMMMMMMMMMMMMMMd      Brightness (Studio Display): 62%
     ;KMMMMMMMWXXWMMMMMMMk.       Brightness (Studio Display): 62%                                                                                        3ms
       "cooc*"    "*coo'"         DE: Aqua                                                                                                                0ms
                                  WM: Quartz Compositor                                                                                                   0ms
                                  WM Theme: Multicolor (Light)                                                                                            0ms
                                  Theme: Not supported on this platform                                                                                   0ms
                                  Icons: Not supported on this platform                                                                                   0ms
                                  Font: .AppleSystemUIFont [System], Helvetica [User]                                                                     2ms
                                  Cursor: Fill - Black, Outline - White (32px)                                                                            0ms
                                  Wallpaper: /System/Library/Desktop Pictures/Studio Color.heic                                                           2ms
                                  Terminal: iTerm 3.4.19                                                                                                  0ms
                                  Terminal Font: JetBrainsMono-Regular_Medium (13pt)                                                                      0ms
                                  CPU: Apple M2 Max (12)                                                                                                  0ms
                                  CPU Usage: 15%                                                                                                          0ms
                                  GPU: Apple M2 Max (30)                                                                                                  2ms
                                  Memory: 24.57 GiB / 64.00 GiB (38%)                                                                                     0ms
                                  Swap: Disabled                                                                                                          0ms
                                  Disk (/): 432 GiB / 2 TiB (23%)
                                  Disk (/Volumes/Backups): 1.51 GiB / 4 TiB (0%)
                                  Disk (/Volumes/Defiant CC): 1 TiB / 2 TiB (70%) [Removable]
                                  Disk (/Volumes/Plex): 5 TiB / 14 TiB (36%)
                                  Disk (/Volumes/lgw4): 5 TiB / 14 TiB (36%)                                                                             85ms
                                  Battery: Querying MaxCapacity failed                                                                                    0ms
                                  Power Adapter: not connected                                                                                            0ms
                                  Media Player: MRMediaRemoteGetNowPlayingInfo() failed                                                                   3ms
                                  Media: MRMediaRemoteGetNowPlayingInfo() failed                                                                          0ms
                                  Public IP: xxx.xx.xx.xxx                                                                                              144ms
                                  Local IP (en0): xx.x.x.xxx
                                  Local IP (en1): xx.x.x.xxx                                                                                              0ms
                                  Wifi: xxxxxxxxxxxxxxxx - 802.11ac (Wi-Fi 5) - WPA3 Personal                                                             48ms
                                  Date & Time: 2023-07-04 16:37:58                                                                                        0ms
                                  Locale: en_US.UTF-8                                                                                                     0ms
                                  Vulkan: dlopen libvulkan.dylib failed                                                                                   1ms
                                  OpenGL: 4.1 Metal - 83.1                                                                                               10ms
                                  OpenCL: 1.2                                                                                                             1ms
                                  Users: lgw4                                                                                                             0ms
                                  Bluetooth 1: xxxxxxxxxxxxxxxxxxxxxxxxxxx
                                  Bluetooth 2: xxxxxxxxxxxxxxxxxxxxxxxxxxx                                                                               67ms
                                  Sound: No active sound devices found                                                                                   37ms
                                  Gamepad: No devices detected                                                                                            0ms
                                  Weather: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx                                                606ms
                                                                                                                                                          0ms
                                  ████████████████████████
                                  ████████████████████████                                                                                                0ms
```

Output of `fastfetch --list-features`:
```
threads
vulkan
imagemagick7
chafa
zlib
sqlite3
```

## If fastfatch crashed

Paste the stacktrace here. You may get it with:

```
$ gdb /path/to/fastfetch
$ run
$ bt
```

If you are able to identify which module crashed, the strace can be helpful too

```
$ strace /path/to/fastfetch --multithreading false --structure {MODULE} --pipe
```

If you cannot do the instructions above, please upload the core dump file:

## If my image logo didn't show / work

* The image protocol you used: 
* The terminal you used: 
* Upload the image file here, or paste the image URL: 
* Does it work with `--logo-width {WIDTH} --logo-height {HEIGHT}`?

**Comment by CarterLi** [comment]
Fixed in dev branch. BTW, did you use two displays?

**Comment by lgw4** [comment]
@CarterLi I confirm the fix in 1.12.2. Thank you for the quick fix.
Yes, I use two (identical) displays.
